Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTableView.cpp

```diff
@@ -148,6 +148,25 @@ namespace AzToolsFramework
         {
         }
 
+        void AssetBrowserTableView::mouseMoveEvent(QMouseEvent* mouseEvent)
+        {
+            QTableView::mouseMoveEvent(mouseEvent);
+
+            QModelIndex hoveredIndex = indexAt(mouseEvent->pos());
```

Comment:
How does this play out if the user is typing at the same time as selecting a row? Would it be possible that the hovered index could be out of range in that case or incorrect?

Reply:
Just tested this. It works fine. `IndexAt` will check if it is a valid index (row >= 0 and column >= 0), if not it will just return `QModelIndex`